src/DMXSerial2.cpp

```diff
@@ -805,6 +814,78 @@ void DMXSerialClass2::_processRDMMessage(byte CmdClass, uint16_t Parameter, bool
 
 // ADD: PARAMETER_DESCRIPTION
 
+    } else if (Parameter == SWAPINT(E120_SENSOR_DEFINITION)) { // 0x0200
+      if (CmdClass == E120_GET_COMMAND) {
+        if (_rdm.packet.DataLength != 1) {
+          // Unexpected data
+          nackReason = E120_NR_FORMAT_ERROR;
+        } else if (_rdm.packet.SubDev != 0) {
+          // No sub-devices supported
+          nackReason = E120_NR_SUB_DEVICE_OUT_OF_RANGE;
+        } else {
+          uint8_t sensorNr = _rdm.packet.Data[0];
+          if (sensorNr >= _initData->sensorsLength) {
```

Comment:
> but you ought to ignore a sensor 255

Isn't this exactly what is happening? If the user should have passed 255 sensors, sensorsLength would also be 255. If then sensor 255 is requested, the library replies with E120_NR_DATA_OUT_OF_RANGE instead of the sensor data, effectively ignoring it.
Also, for any smaller number of defined sensors 255 will result in E120_NR_DATA_OUT_OF_RANGE.

Sorry, but I still do not see the case where this is problematic...